### ISSUE TITLE
Administrative section

### ISSUE THREAD

**Issue by pjotrsavitski** [centre-for-educational-technology/EzTest#33]
Consider a need to create a basic administrative section.

Main use case could be some basic user management and simple statistics about the system as such.

Statistics: how many users, how many questions, how many groups, how many tests, how many test submissions (maybe something more complicated).

All the needs should be described, this would also require some basic user roles within the system (the simplest solution would be to either flag an administrator account in some way or add some kind on role name or number identifier to each user account).

**Comment by vtomberg** [comment]
> Consider a need to create a basic administrative section.

Agree. It should be visible only for the users who are logged in with the administrative password.

For now I see three roles: Admin, Instructor, Student.
